src/azul/reindexer.py

```diff
@@ -0,0 +1,137 @@
+#! /usr/bin/env python3
+
+"""
+Command line utility to trigger indexing of bundles from DSS into Azul
```

Comment:
This must remain in the script.

Reply:
I must have missed this comment. I removed it